examples/panic.rs

```diff
@@ -1,9 +1,10 @@
+// Triggers the panic handler. Should make all LEDs flash.
+
 #![no_std]
 
 use libtock::result::TockResult;
```

Comment:
Do you think it's necessary to explain how imports work? IMHO I cannot imagine anybody trying to run an example without importing anything from `libtock-rs`. The old panic example was a very special situation.

Reply:
I think there is no use case for using libtock without importing anything from it.

Reply:
We should make sure that an empty program (nothing in `main()`'s body) still compiles, which I believe necessitates an `extern crate libtock;`. Otherwise people will have trouble getting started.